src/components/ToggleButton/ToggleButton.tsx

```diff
@@ -5,6 +5,7 @@ import { Button } from '../Button/Button';
 export interface ToggleButtonProps {
 	icon: 'heart' | 'bookmark';
 	active: boolean;
+	ariaLabel: string;
```

Comment:
Optional? Is enkel nodig voor buttons die geen label hebben.

Reply:
togglebuttons hebben geen label

Reply:
Goed punt, my bad 😅